mathext/ell_complete.go

```diff
@@ -0,0 +1,157 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import (
+	"math"
+)
+
+// CompleteK computes the complete elliptic integral of the 1st kind, 0≤m≤1.
+// It returns math.NaN() if m is not in [0,1].
+//
+//	K(m) = \int_{0}^{\pi/2} d\theta / {\sqrt{1-m{\sin^2\theta}}}
```

Comment:
Could you please put `d\theta` at the end? I feel that without mathematical typesetting it would be clearer.